core/block_test.go

```diff
@@ -145,30 +91,6 @@ func TestBlockHash(t *testing.T) {
 			chain:  utils.Mainnet,
 			name:   "Block 192 with Failing l1 handler transaction version 1",
 		},
-		// "https://external.integration.starknet.io/feeder_gateway/get_block?blockNumber=283364"
-		{
-			number: 283364,
-			chain:  utils.Integration,
-			name:   "Block 283364 with Declare v2",
```

Comment:
same, specific tx type with version

Reply:
These one I looked for but found only version v3  - let me run my scripts over the weekend 🤔

Reply:
add Sepolia block `#7`